lib/errors/spec.js

```diff
@@ -74,6 +74,21 @@ module.exports = [{
       name: 'invalidName',
       message: 'Invalid Name'
     }]
+  }, {
+    name: 'Trigger',
+    message: 'Internal Error on Proposal {0}',
+    errors: [{
+      name: 'invalidAddress',
+      message: 'Invalid Address'
+    },
+    {
+      name: 'invalidP2SHAddress',
```

Comment:
I don't think P2SH are allowed (yet), but might ask @UdjinM6 to be sure. I know that Sentinel will reject them, not sure about Core.